stdlib/asyncio/sslproto.pyi

```diff
@@ -106,7 +106,7 @@ class SSLProtocol(protocols.Protocol):
             loop: events.AbstractEventLoop,
             app_protocol: protocols.BaseProtocol,
             sslcontext: ssl.SSLContext,
-            waiter: futures.Future,
+            waiter: futures.Future[Any],
```

Comment:
This isn't failing because the current pyright test runs only for Python 3.9. The next version of pyright adds support for specifying different versions of Python from the command line, so we'll update the test to iterate over multiple versions and platforms. Without this change, the test will fail for <3.7.